docs/tests-creating.md

```diff
@@ -21,15 +21,15 @@ To create your first Postman collection in Testkube, export your collection into
 
 Right click on your collection name:
 
-![create postman collection step 1](img/test-create-1.png)
+![create postman collection step 1](img/script-create-1.png)
```

Comment:
We don't have `script` anymore in our terms. Either it should be new test-something image or rename exsiting image, if it's already with tests

Reply:
I can rename the image file.  I left them the same in case they were used by something I was not aware of.

Reply:
I changed the image file names to test-create... and updated tests-creating.md.